### PR TITLE
Fix a bug caused by PyTorch instance_norm when the input shape is [1,1,1,2]

### PR DESCRIPTION
**Pull request by haoyang9804**
Fix [Issue 15282](https://github.com/apache/tvm/issues/15282)

### CODE REVIEW

**Comment by haoyang9804** [comment]
cc @vvchernov @echuraev

**Review by jikechao — APPROVED** [review]
LGTM, thanks for your fixing.

File: python/tvm/relay/frontend/pytorch.py [hunk 1]
```diff
@@ -4424,7 +4424,10 @@ def _create_typed_const(data, dtype):
     dtype should be a TVM dtype"""
 
     if dtype == "float64":
-        typed_data = _expr.const(np.float64(data), dtype=dtype)
+        if len(data) == 1:
+            typed_data = _expr.const(np.array([np.float64(data)]), dtype=dtype)
+        else:
+            typed_data = _expr.const(np.float64(data), dtype=dtype)
```
Comment (echuraev) [inline python/tvm/relay/frontend/pytorch.py:4430]: I read your updates in the issue, thank you for detailed description. Here it looks like a workaround above numpy bug. I think that the best way to fix this issue is to create an issue or a PR in the numpy repo with fix.

Currently, this implementation won't work after fixing the bug in numpy. My suggestion is to create an issue or PR with fix to numpy repository and at the same time change the current implementation in the following way:
```suggestion
        typed_data = np.float64(data)
        if np.isscalar(typed_data):
            typed_data = _expr.const(np.array([typed_data]), dtype=dtype)
        else:
            typed_data = _expr.const(typed_data, dtype=dtype)
```
This code should work also after fixing the bug in Numpy.

**Review by echuraev — COMMENTED** [review]
Thank you for your PR!

File: python/tvm/relay/frontend/pytorch.py (hunk 1 above)
Comment (haoyang9804) [reply python/tvm/relay/frontend/pytorch.py:4430]: Thanks! I have submitted a issue to Numpy community [here](https://github.com/numpy/numpy/issues/24661).

Comment (echuraev) [reply python/tvm/relay/frontend/pytorch.py]: Thank you! In this case, something like `typed_data = _expr.const(np.asarray(data, dtype="float64"), dtype=dtype)` might be the best solution for this problem.

File: python/tvm/relay/frontend/pytorch.py [hunk 2]
```diff
@@ -4424,7 +4424,11 @@ def _create_typed_const(data, dtype):
     dtype should be a TVM dtype"""
 
     if dtype == "float64":
-        typed_data = _expr.const(np.float64(data), dtype=dtype)
+        typed_data = np.float64(data)
+        if np.isscalar(typed_data):
+            typed_data = _expr.const(np.asarray(data, dtype="float64"), dtype=dtype)
+        else:
+            typed_data = _expr.const(typed_data, dtype=dtype)
```
[the remark this answers is not in the document]
Comment (haoyang9804) [reply python/tvm/relay/frontend/pytorch.py]: True. Thanks for your reminder. Have changed the patch

**Review by echuraev — APPROVED** [review]
LGTM. Thanks!

**Pull request merged**